bookmarklet.js

```diff
@@ -2677,11 +2677,11 @@ SherdBookmarklet = {
                 asset.sources.poster;
             var newAsset;
             if (img) {
-                newAsset = self.elt(null,'img','sherd-image',{src:img,style:'max-width:215px;max-height:150px',height:null});
```

Comment:
I think the width/height was added to prevent very large images from taking up too much screen real-estate. Maybe something more like: max-width: 215px; height: auto; would resolve the aspect ratio issue and control the display?